web/params.go

```diff
@@ -69,6 +69,7 @@ type Params struct {
 	NotAssociatedToGroup   string
 	ExcludeDefaultChannels bool
 	GroupIDs               string
+	PluginDownloadUrl      string
```

Comment:
Is this worth putting here when it's only going to be used in this one route?

Reply:
I agree. Do you think we should be using a json body for this request, or do query params suffice?